app/views/quick_reference/steps/command/_agents.md.erb

```diff
@@ -0,0 +1,7 @@
+### agents
```

Comment:
are we putting titles in this? if so why are we rendering titles in the frontend component?

Reply:
Whoops, good pickup! I've stripped the titles out of the markdown content